modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/CalciteQueryProcessorTest.java

```diff
@@ -1049,6 +1049,27 @@ public void quantifiedCompTest() throws InterruptedException {
             .returns(13d)
             .check();
 
+        assertQuery(client, "select salary from account where salary > SOME (10, 11) ORDER BY salary OFFSET 1")
```

Comment:
Let's move limit and offset checks to `LimitOffsetTest` class (also move `LimitOffsetTest` to `integration` package and inherit from `AbstractBasicIntegrationTest`)

Reply:
Yep. Test `testLimitOffset()` looks enough.

Reply:
Moved. But I doubt we need extend AbstractBasicIntegrationTest

Reply:
Hm. LimitOffsetTest seems to have enough limit/offset tests. Moved.